packages/trpc/server/routers/viewer/eventTypes/create.handler.ts

```diff
@@ -84,6 +86,7 @@ export const createHandler = async ({ ctx, input }: CreateOptions) => {
     };
     data.schedulingType = schedulingType;
   }
+  await removePreviousSlug({ userId, teamId, previousSlug: slug });
```

Comment:
when creating a new `event-type` remove any `previousSlug` that equals the `slug` in any of the `event-type` of the given user/team.